### PR TITLE
feat: add filename lint dockerfile

### PR DESCRIPTION
**Pull request by ArielHAlba**
Add dockerfile for filename linting

### CODE REVIEW

File: filename-lint/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN git clone --branch v2.1.0 https://github.com/bloom42/flint.git 
+
+WORKDIR /flint
+COPY ./ ./
+RUN make build_static
+
+####################################################################################################
+## Image
+####################################################################################################
```
Comment (ferhatelmas) [inline filename-lint/Dockerfile]: let's remove comment

File: filename-lint/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN apt update && apt install -y make musl-dev musl-tools pkg-config
+
+RUN useradd -ms /bin/sh bloom
+
+RUN git clone --branch v2.1.0 https://github.com/bloom42/flint.git 
+
+WORKDIR /flint
+COPY ./ ./
```
Comment (ferhatelmas) [inline filename-lint/Dockerfile]: why do you need this ?

File: filename-lint/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM rust:1.33-stretch AS builder
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN apt update && apt install -y make musl-dev musl-tools pkg-config
+
+RUN useradd -ms /bin/sh bloom
```
Comment (ferhatelmas) [inline filename-lint/Dockerfile]: why is user needed ?

File: filename-lint/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# docker-filename-lint
+
+> file name lint in docker image
+## License
+
+MIT © [AgFlow](https://www.agflow.com)
```
Comment (ferhatelmas) [inline filename-lint/README.md]: rm license, repo has a general one

File: filename-lint/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+RUN git clone --branch v2.1.0 https://github.com/bloom42/flint.git 
+
+WORKDIR /flint
+COPY ./ ./
+RUN make build_static
+
+####################################################################################################
+## Image
+####################################################################################################
+FROM ubuntu:18.10
```
Comment (ferhatelmas) [inline filename-lint/Dockerfile]: why not alpine ?

Comment (ArielHAlba) [reply filename-lint/Dockerfile]: yes alpine its better, less size, fast!

Comment (ArielHAlba) [reply filename-lint/Dockerfile]: however I need bash for mapfile operations, i can use an alpine one and install bash on it or directly the ubuntu one

Comment (ferhatelmas) [reply filename-lint/Dockerfile]: ubuntu is good then, pretty optimized for bash.

File: flint/README.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# docker-filename-lint
```
Comment (ferhatelmas) [inline flint/README.md]: expand / explain more and also write only flint in title

File: flint/README.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# flint
+
+> Minimal ubuntu docker image with [flint](https://github.com/bloom42/flint) 
+v2.1.0 binary ready
```
Comment (ferhatelmas) [inline flint/README.md]: version is something we need to maintain - better leave out

**Pull request merged**